### DISCUSSION TITLE
Getting 404 error

### DISCUSSION THREAD

**Discussion by mohamed-ghouse** [community/community#67741] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Team,
 I have created a repository with the name mohamed-ghouse and wanted to use Pages.
The repository has index.html. However, when I access the site using https://mohamed-ghouse.github.io , I am getting 404 error. I have waited for more than an hour thinking the DNS propagation changes.
No luck.
I tried with the steps mentioned in https://docs.github.com/en/pages/getting-started-with-github-pages/troubleshooting-404-errors-for-github-pages-sites.

Still no luck.

Please let me know anything that I am missing.

Thanks,
Mohamed

**Comment by Abhiram-J** [comment, marked as answer]
Hi Mohamed,
Inorder to access the site https://mohamed-ghouse.github.io/ , you need to have the repository name as mohamed-ghouse.github.io

**Comment by mohamed-ghouse** [comment]
Thanks! My bad.
I was creating a repository with mohamed-ghouse only instead of mohamed-ghouse.github.io

It works!

Thanks,
Mohamed

**Reply by Abhiram-J** [reply]
Happy to help you !
Can you mark my above response as answer, so that this thread can be closed.
